Test/Unit/Model/Api/OrderManagementTest.php

```diff
@@ -48,875 +77,1964 @@
+                            "unit_price" => [
+                                "amount" => self::PRODUCT_PRICE*100,
+                                "currency" => self::CURRENCY,
+                                "currency_symbol" =>  self::CURRENCY_USD_SYMBOL,
+                            ],
+                            "quantity" => self::QUOTE_PRODUCT_QTY,
+                            "type" => "physical"
+                        ]
+                    ],
+                    'billing_address' =>
+                    [
+                        "street_address1" => $addressInfo["street_address1"],
+                        "locality"        => $addressInfo["locality"],
+                        "region"          => $addressInfo["region"],
+                        "postal_code"     => $addressInfo["postal_code"],
+                        "country_code"    => $addressInfo["country_code"],
+                        "country"         => $addressInfo["country"],
+                        "name"            => $addressInfo["name"],
+                        "first_name"      => $addressInfo["first_name"],
+                        "last_name"       => $addressInfo["last_name"],
+                        "phone_number"    => $addressInfo["phone_number"],
+                        "email_address"   => $addressInfo["email_address"],
+                    ],
+                    "shipments" => [
+                        [
+                            'cost'             => self::QUOTE_SHIPPING_TOTAL*100,
+                            'tax_amount'       => self::QUOTE_SHIPPING_TAX_TOTAL*100,
+                            "shipping_address" => [                                
+                                "street_address1" => $addressInfo["street_address1"],
+                                "locality"        => $addressInfo["locality"],
+                                "region"          => $addressInfo["region"],
+                                "postal_code"     => $addressInfo["postal_code"],
+                                "country_code"    => $addressInfo["country_code"],
+                                "country"         => $addressInfo["country"],
+                                "name"            => $addressInfo["name"],
+                                "first_name"      => $addressInfo["first_name"],
+                                "last_name"       => $addressInfo["last_name"],
+                            ],
+                            'service'          => 'Flat Rate - Fixed',
+                            'reference'        => 'flatrate_flatrate',
+                            "shipping_method"  => "unknown"
+                        ]
+                    ],
+                    'metadata'        => [
+                        'immutable_quote_id' => $immutableQuoteId,
+                    ],
+                ]
+            ]
+        ];
+        
+        return $requetData;
+    }
+    
+    
+    
+    private function createProduct()
```

Comment:
@johnmccombs1 already created some of helpers. Anyway, we need to move those method out to reuse them in other test classes

Reply:
I move some functions to class TestUtils and refactor some others to resue the existing util functions.